src/components/AVPlayer/AVPlayer.js

```diff
@@ -879,4 +886,12 @@ class AVPlayer extends Component {
   }
 }
 
-export default withNamespaces()(withMediaProps(withRouter(AVPlayer)));
+export const mapState = () => ({});
+
+export const mapDispatch = dispatch => (
```

Comment:
Why to export it? Is it used somewhere else?

Reply:
Fixed.